Source/Manager/User/User.swift

```diff
@@ -292,6 +292,38 @@ public class User: UserProtocol {
         isPersistent = true
     }
 
+    func storeOnDevice(key: String) throws {
+        do {
+            let tokens = try UserTokensStorage(accessGroup: clientConfiguration.accessGroup).loadTokens()
+            let keyPrefix = "old-sdk-app-transfer-"
+
+            if let encoded = try? JSONEncoder().encode(tokens) {
+                UserDefaults.standard.set(encoded, forKey: keyPrefix + key)
+            }
+        } catch {
+            log(level: .error, from: self, "failed to storeOnDevice tokens: \(error)", force: true)
+        }
+    }
+
+    func loadFromDeviceToKeychain(key: String) throws {
+        let keyPrefix = "old-sdk-app-transfer-"
+
+        if let tokenData = UserDefaults.standard.object(forKey: "old-sdk-app-transfer-" + key) as? Data,
```

Comment:
you have this `"old-sdk-app-transfer-"` stored in `keyPrefix` but you are not using it

Reply:
Is it not user on line 219?

Reply:
320? Yes, but it can be used on line 311 too. So we prevent a typo bug.

Reply:
Thats better. I added it as static 🪚

Reply:
resolving this conversation